Loenn/entities/rusty_lamp.lua

```diff
@@ -0,0 +1,53 @@
+local utils = require("utils")
+
+local rustyLamp = {}
+
+rustyLamp.name = "FactoryHelper/RustyLamp"
+
+rustyLamp.fieldInformation = {
+    strobePattern = {
+        editable = false,
+        options = {
+            ["None"] = "None",
+            ["Flicker On"] = "FlickerOn",
+            ["Light Flicker"] = "LightFlicker",
+            ["Turn Off, Flicker On"] = "TurnOffFlickerOn"
```

Comment:
There's a `FlickerOff` routine in the code that's not in the Ahorn plugin either. Should probably add them to both?